testsuite/forge/src/backend/local/swarm.rs

```diff
@@ -376,11 +379,16 @@ impl LocalSwarm {
     }
 
     pub fn validators(&self) -> impl Iterator<Item = &LocalNode> {
-        self.validators.values()
+        // sort by id to keep the order stable:
+        let mut validators: Vec<&LocalNode> = self.validators.values().collect();
+        validators.sort_by_key(|v| v.name().parse::<i32>().unwrap());
+        validators.into_iter()
     }
 
     pub fn validators_mut(&mut self) -> impl Iterator<Item = &mut LocalNode> {
-        self.validators.values_mut()
+        let mut validators: Vec<&mut LocalNode> = self.validators.values_mut().collect();
```

Comment:
Instead of sorting it every time in this call, can we keep the sorted values in `self.validators`

Reply:
validators is a hashmap.

I can store two fields, referring to same nodes, validators_map and validators_sorted if you prefer?

Reply:
I see. Its fine in that case.